mechanize.gemspec

```diff
@@ -56,6 +56,7 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency "http-cookie",          [ "~> 1.0" ]
   spec.add_runtime_dependency "nokogiri",             [ "~> 1.6" ]
   spec.add_runtime_dependency "ntlm-http",            [ ">= 0.1.1", "~> 0.1"   ]
+  spec.add_runtime_dependency "webrick",              [ "~> 1.7" ]  if RUBY_VERSION >= "3.0"
```

Comment:
`if RUBY_VERSION >= "3.0"` isn't needless.

c.f. https://docs.rubocop.org/rubocop/cops_gemspec.html#gemspecrubyversionglobalsusage

Reply:
wouldn't it just install as a gem dependency as needed?

Reply:
No. 

> because value of the constant is determined by rake release

This is all. 

`RUBY_VERSION` in `gemspec` is used at `rake release` by maintainer. (**NOT** `gem install` or `bundle install` by user)

Reply:
Thank you @sue445, TIL :+1: